OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -279,14 +279,25 @@ public CPos ViewToWorld(int2 view)
 		}
 
 		/// <summary>Returns an unfiltered list of all cells that could potentially contain the mouse cursor.</summary>
-		IEnumerable<MPos> CandidateMouseoverCells(int2 world)
+		public IEnumerable<MPos> CandidateMouseoverCells(int2 world)
```

Comment:
We'll need to get rid of all the `if (map.Grid.Type == MapGridType.RectangularIsometric)` as part of the move to mod-defined map formats.

IMO it would make sense to move this method to `Map.cs` now in preparation for that.

Reply:
Or rather the function that consumes it. `ViewToWorld`

Reply:
Tbf it isn't very simple. It should be reserved for a separate PR